web-registry/src/components/templates/ProjectDetails/ProjectDetails.utils.tsx

```diff
@@ -210,12 +211,9 @@ export function getDisplayParty(
   metadata?: AnchoredProjectMetadataBaseLD,
   party?: Maybe<PartyFieldsFragment>,
 ): Party | undefined {
-  const showOnProjectPage = metadata?.[role]?.['regen:showOnProjectPage'];
```

Comment:
this was actually from some legacy logic, I believe now the logic with public profile and the latest version of the roles form is rather to display the project stakeholder on the project page as soon as it's defined
cc/ @erikalogie